### ISSUE TITLE
Clion doesn't use rustfmt when "Use rustfmt instead of the built-in formatter" is selected

### ISSUE THREAD

**Issue by jrg1k** [intellij-rust/intellij-rust#10835]
## Environment

* **IntelliJ Rust plugin version:** 0.4.200.5421-232
* **Rust toolchain version:** stable-x86_64-unknown-linux-gnu - rustc 1.72.0
* **IDE name and version:** CLion 2023.2
* **Operating system:** Fedora Linux 38 (Thirty Eight)

## Problem description

Clion doesn't use rustfmt when "Use rustfmt instead of the built-in formatter" is selected. On save works but the ReformatFile action does not trigger rustfmt, it triggers the built in formatter.

## Steps to reproduce

This code is left as-is by the built-in formatter:

```rust
struct Connection<S> where S: AsyncRead + AsyncWrite,
{
    stream: S,
    buffer: VecDeque<u8>,
}
```

rustfmt formats it like this:

```rust
struct Connection<S>
where
    S: AsyncRead + AsyncWrite,
{
    stream: S,
    buffer: VecDeque<u8>,
}
```

**Comment by Maksim20023** [comment]
Hi! Thanks for reaching us.
This is a known issue for us and has already been reported. Please take a look at #10752. You can subscribe to it to stay informed and vote (click the emoji button in the bottom-left corner of the GitHub post). When we have new information, we will add a reply to that ticket. 
If you have any further suggestions, please feel free to reach out.
